packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3616,6 +3637,30 @@ export default class Meeting extends StatelessWebexPlugin {
       });
   }
 
+  /**
+   * Connects to low latency mercury and reconnects if the address has changed
+   * @param {String} datachannelUrl
+   * @returns {Promise}
+   */
+  async ensureLLMConnected() {
```

Comment:
the name is a bit misleading because this function also just disconnects LLM if we are not joined, but not sure what a better name would be without making it very long. Maybe at least mention the disconnection in the comments?